textual/copyright.txt

```diff
@@ -0,0 +1 @@
+Developed by Imran Ismail
```

Comment:
Is this file really necessary? Couldn't we use same comment pattern as the other ones?